fyt/reports/tests.py

```diff
@@ -704,20 +704,32 @@ def test_external_bus_riders(self):
     def test_gear_requests(self):
         gear1 = mommy.make(Gear, trips_year=self.trips_year)
         gear2 = mommy.make(Gear, trips_year=self.trips_year)
+
         request1 = mommy.make(
             GearRequest,
             trips_year=self.trips_year,
             incoming_student__trips_year=self.trips_year,
             additional='teddy bear',
         )
         request1.gear.set([gear1])
+
         request2 = mommy.make(
             GearRequest,
             trips_year=self.trips_year,
             volunteer__trips_year=self.trips_year,
             volunteer__status=Volunteer.LEADER,
         )
         request2.gear.set([gear2])
+        request2.provided.set([gear2])
+
+        # request3 = mommy.make(
+        #     GearRequest,
+        #     trips_year=self.trips_year,
+        #     volunteer__trips_year=self.trips_year,
+        #     volunteer__status=Volunteer.LEADER,
+        # )
+        # request3.gear.set([gear1, gear2])
+        # request3.provided.set([gear1])
```

Comment:
Thinking more about this, I think the failure happens because the CSV is ordered by incoming students, then volunteers. When you add another volunteer, the list becomes non-deterministic. You could override the applicant names with `volunteer__applicant__name="A"` (or something like that) to get the desired result.

Reply:
Yup that explains it, because the original case has one volunteer and one trippee. Adding a second volunteer is the problem. I'll uncomment the test case.